pkg/sql/colexec/multi_col/group_concat/group_concat.go

```diff
@@ -247,10 +248,9 @@ func (gc *GroupConcat) Fill(groupIndex int64, rowIndex int64, rowCount int64, ve
 		s, _ := VectorToString(vecs[i], int(rowIndex))
 		res_row += s
 		// prefix length + data
-		length := uint16(len(s))
-		// WTF is the following crap?
-		// insert_row += string(unsafe.Slice((*byte)(unsafe.Pointer(&length)), 2)) + s
-		insert_row += unsafe.String((*byte)(unsafe.Pointer(&length)), 2) + s
+		bs := make([]byte, 2)
```

Comment:
is uint16 big enough?    len(s) be more than 64K?

Reply:
The old code uses uint16 so I assume it's OK.

Reply:
That is what I think too so until it is not ok, then boom.     @m-schen

Reply:
@reusee how about add an assert?   If it does not fit in uint16 throw an error.

Reply:
Done

Reply:
I don't know what does these codes do. Just let it go.
If panic, it can also prove that old result was wrong.

Reply:
I don't know what does these codes do. Just let it go.
If panic, it can also prove that old result was wrong. and we fix it.